test/controllers/ErrorPagesHandlerSpec.scala

```diff
@@ -65,7 +65,7 @@ class ErrorPagesHandlerSpec extends BaseSpec {
       val title = doc.select("title").text()
       title must include("Bad request - 400")
 
-      val heading = doc.select(".h1-heading").text()
+      val heading = doc.select(".govuk-heading-l").text()
```

Comment:
the original idea was to have checkSomethingNew + checkSomething together so no test would be broken. Once everything was migrated big refactor to remove New from all methods